pkg/deploy/generator/scripts/rpVMSS.sh

```diff
@@ -640,4 +653,4 @@ done
 
 echo "rebooting"
 restorecon -RF /var/log/*
-(sleep 120; reboot) &
+(sleep 30; reboot) &
```

Comment:
Hey David, thanks for this work. I am worried about lowering this sleep value too much as we are colouring outside the lines a little bit by issuing a reboot command through this script already. We basically create a little bit of a race where we expect the MS WAagent to properly finish and report status before this reboot goes through. 30 seconds may not be enough so if this doesn't cause any issues I'd like to keep it at 120, but open to your comments :)

Reply:
The only reason we ended up with a race condition was because the `-x WALinuxAgent` was removed. Subsequently I found that we also need `-x WALinuxAgent-udev`, after many rounds of manual testing. Since those have been added I have never seen any agent issues in any of the tests I have performed.

There is another more important reason why we don't want the sleep to be so high and that is Load Balancer health checks. If the service is up for long enough, after the bootstrap, the ARO-RP can start processing requests and then be abruptly rebooted. To avoid this, keeping the sleep at 30 seconds makes the most sense IMO.